http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzureTestUtils.scala

```diff
@@ -51,6 +57,21 @@ object AzureTestUtils extends MockitoSugar {
         .errorReport(new ErrorReport())
     }
 
+    // Create storage container
+    when {
+      api.createAzureStorageContainer(any, ArgumentMatchers.eq(workspaceId.value))
+    } thenAnswer { invocation =>
+      val requestBody = invocation.getArgument[CreateControlledAzureStorageContainerRequestBody](0)
+      new CreatedControlledAzureStorageContainer().resourceId(UUID.randomUUID())
```

Comment:
unsure why you are retrieving the `requestBody` but then creating a new resourceId here instead of using the one possibly provided in the body? Seems like you can just remove line 64

Reply:
good point, removed